src/Temporalio.Extensions.Hosting/Temporalio.Extensions.Hosting.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net462;net8.0</TargetFrameworks>
```

Comment:
I saw the comment on .NET 8 vs .NET 6, but why wouldn't publishing as 6.0 work for all post 6.0 users?

Reply:
Purely opinionated choice, why start with NET6.0, when NET8.0 is the current recommendation for LTS. Vice versa, had .NET9.0 been released, I would have chosen NET8.0. If we already were supporting NET6.0 as well, I would have added NET8.0 and kept NET6.0. They are different libraries, so if we want it to work with NET6 + NET8 it's better to package them that way. Compatibility not withstanding.

In general, I target the latest LTS, on the off chance there is a breaking behavior change between NET6/NET8, we can ensure the expected behavior is that of NET8. It's entirely choice though.

Reply:
> why start with NET6.0, when NET8.0 is the current recommendation for LTS

I think it's because you might as well choose the lowest common denominator your features require regardless of LTS if it's going to work for later ones. Same for the precompiler directive. If we add .NET-8 specific stuff, we can publish a .NET 8 version, but there seems to be 0 benefit in choosing .NET 8 for this library when .NET 6 provides a superset of benefits.

Reply:
Yeah, happy to make it NET6. Another reason I try and always bump early, is that while the [breaking changes](https://learn.microsoft.com/en-us/dotnet/core/compatibility/8.0) are listed (thanks Microsoft!) it can be a bit mental effort to review all the changes and predict what's up when a user isn't working, especially leapfrogging versions

Reply:
:+1: Totally happy if/when we see that to add `net8.0` here

Reply:
I will know this for future PRs too. Thanks for taking the time to chat!